DOCS-1247-PGD-6.0-Docs (#6763)
EnterpriseDB/docs: product_docs/docs/pgd/6/reference/tables-views-functions/functions.mdx

If a CAMO transaction is in progress, `transaction_id` is updated to show
the assigned transaction id. You can query this parameter only by using

> using `PQparameterStatus` or equivalent, and it is not accessible in SQL.

Reviewer: ```suggestion
using `PQparameterStatus` or equivalent, and it isn't accessible in SQL.
```

with `bdr.ddl_replication = off`, since the commands are already being sent
to all of the nodes in the cluster.

From PGD 6 onwards, this function also sets `bdr.xact_replication=off` on the

> connection, to ensure that transactions only run locally when the command is

Reviewer: ```suggestion
connection to ensure that transaction run locally only when the command is
```

with `bdr.ddl_replication = off` to avoid replication issues when the same
replicated DDL command is sent to multiple nodes.

> From PGD 6 onwards, this function also sets `bdr.xact_replication=off` on the

Reviewer: ```suggestion
In PGD 6 and later, this function also sets `bdr.xact_replication=off` on the
```

> connection, to ensure that transactions only run locally when the command is

Reviewer: ```suggestion
connection to ensure that transactions run locally only when the command is
```

with `bdr.ddl_replication = off` to avoid replication issues when the same
replicated DDL command is sent to multiple nodes.

> From PGD 6 onwards, this function also sets `bdr.xact_replication=off` on the

Reviewer: ```suggestion
In PGD 6 and later, this function also sets `bdr.xact_replication=off` on the
```

> connection, to ensure that transactions only run locally when the command is

Reviewer: ```suggestion
connection to ensure that transactions run locally only when the command is
```

#### Note

> `bdr.create_commit_scope` replaces the deprecated [`bdr.add_commit_scope`](/pgd/latest/reference/tables-views-functions/functions#bdradd_commit_scope) function. Unlike `add_commit_scope`, it does not silently overwrite existing commit scopes when the same name is used. Instead, an error is reported.

Reviewer: ```suggestion
`bdr.create_commit_scope` replaces the deprecated [`bdr.add_commit_scope`](/pgd/latest/reference/tables-views-functions/functions#bdradd_commit_scope) function. Unlike `add_commit_scope`, it doesn't silently overwrite existing commit scopes when the same name is used. Instead, an error is reported.
```
